crates/re_data_ui/src/lib.rs

```diff
@@ -29,6 +29,27 @@ pub use crate::image::{
 pub use component_ui_registry::create_component_ui_registry;
 pub use image_meaning::image_meaning_for_entity;
 
+/// Show this component in the UI.
+pub fn is_component_visible_in_ui(component_name: &re_types::ComponentName) -> bool {
+    const HIDDEN_COMPONENTS: &[&str] = &["rerun.components.InstanceKey"];
+    !HIDDEN_COMPONENTS.contains(&component_name.as_ref())
+}
+
+/// Is this an indicator component for an archetype?
+pub fn is_indicator_component(component_name: &re_types::ComponentName) -> bool {
+    component_name.starts_with("rerun.components.") && component_name.ends_with("Indicator")
```

Comment:
This makes me wonder if we should consider putting these in their own namespace like `rerun.indicators`?